src/legacy/core_plugins/tagcloud/public/tag_cloud_vis_options.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { EuiPanel } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { ValidatedDualRange } from 'ui/validated_range';
+import { VisOptionsProps } from 'ui/vis/editors/default';
+import { SelectOption } from '../../kbn_vislib_vis_types/public/controls/select';
+import { SwitchOption } from '../../kbn_vislib_vis_types/public/controls/switch';
+
+function TagCloudOptions({ stateParams, setValue, vis }: VisOptionsProps) {
+  const handleFontSizeChange = ([minFontSize, maxFontSize]: string[]) => {
+    setValue('minFontSize', minFontSize);
+    setValue('maxFontSize', maxFontSize);
+  };
+  const fontSizeRangeLabel = i18n.translate('tagCloud.visParams.fontSizeLabel', {
+    defaultMessage: 'Font size range (pixels)',
```

Comment:
Just got word from our copy writer, change this to:
```suggestion
    defaultMessage: 'Font size range in pixels',
```

Thanks!

Reply:
Done